components/Report.vue

```diff
@@ -229,6 +232,11 @@
   </div>
 </template>
 <style lang="scss" scoped>
+@media screen and (max-width: 480px) {
```

Comment:
I played with this a lot to try and figure out where to set these breakpoints for different behavior, probably worth just having a short call about it.  The whole app becomes comfortably usable around 860px wide (any narrower, the tables won't scroll or collapse anymore).  Resolutions under 760 tend to mangle the graphs in ways that aren't so good.